packages/aws-library/src/aws_library/s3/client.py

```diff
@@ -4,27 +4,31 @@
 from typing import cast
 
 import aioboto3
-import botocore.exceptions
 from aiobotocore.session import ClientCreatorContext
 from botocore.client import Config
 from models_library.api_schemas_storage import S3BucketName
 from pydantic import AnyUrl, parse_obj_as
 from settings_library.s3 import S3Settings
 from types_aiobotocore_s3 import S3Client
 
-from .errors import S3RuntimeError
+from .errors import s3_exception_handler
 
 _logger = logging.getLogger(__name__)
 
+_S3_MAX_CONCURRENCY_DEFAULT = 10
+
 
 @dataclass(frozen=True)
 class SimcoreS3API:
     client: S3Client
```

Comment:
Q: should this client be protected to avoid it to be used from outside?
same for session, exit_stack ...

Reply:
Let's discuss in person, I will now move forward :+1: